script/init-script.sh

```diff
@@ -0,0 +1,3 @@
+gem install rubocop
```

Comment:
on my laptop this had to be sudo

Reply:
If you update this line to add the `sudo`, I can merge it in :)